hotosm/tasking-manager
# Feature/OSM Download meta data integration

## What does this PR do ? 
- Meta data shown to let user know it's updated date and size.

## Consideration 
-On previous approach , We didn't had any metadata, which was causing user to know last updated date and the actual size of the files.

## How to test ? 
-You can see two text one on side of the download with size of the file i.e. Geojson (2.4mb) and one below the filetype as last modified date as shown in the picture.

![image](https://github.com/hotosm/tasking-manager/assets/37866666/61eb811e-92bb-4941-ba85-e38941620589)
